test/express-rate-limit-test.js

```diff
@@ -307,6 +307,28 @@ describe("express-rate-limit node module", function() {
     );
   });
 
+  it("should send correct ratelimit-limit and ratelimit-remaining", function(done) {
```

Comment:
This is a good start, but it isn't actually checking the new headers, it's just checking the delay length and the old x-ratelimit-* headers (which are on by default) in the goodRequest method. 

Instead of going that route, lets skip the goodRequest method and just create our own request and do our own checks. In that case, the body would look something more like this (I haven't tested this, but it should be about right):

```js
    const limit = 5;
    const windowMs = 60 * 1000; // 60 * 1000 = 1 minute
    const app = createAppWith(
      rateLimit({ windowMs: windowMs, limit: limit, draft_polli_ratelimit_headers: true })
    );
    const expectedRemaining = 4;
    const expectedResetSeconds = 60;
    request(app).get("/");
        .expect("ratelimit-limit", limit.toString())
        .expect("ratelimit-remaining", expectedRemaining.toString())
        .expect("ratelimit-reset", expectedResetSeconds.toString())
        .expect(200, /response!/)
        .end(done);
  }
```

Reply:
Oh, I missed your question asking about amending `goodRequest`. Sorry. I think this is probably a better route anyways.

Reply:
Alright, on it then!